pkg/storage/system_intake_funding_source.go

```diff
@@ -0,0 +1,80 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// UpdateSystemIntakeFundingSources clears and updates the funding sources of a system intake
+func (s *Store) UpdateSystemIntakeFundingSources(ctx context.Context, systemIntakeID uuid.UUID, fundingSources []*models.SystemIntakeFundingSource) ([]*models.SystemIntakeFundingSource, error) {
```

Comment:
There's a few debug statements in this function that can probably be ripped out.

If you really want to, you could always use the logger and log these with a `trace` level if you think they'd be helpful! (Would be worth confirming that they wont show up in Splunk, though)